doc/src/manual/functions.md

```diff
@@ -558,43 +570,91 @@ the arguments of the user function are initialized.
 
 In technical-computing languages, it is common to have "vectorized" versions of functions, which
 simply apply a given function `f(x)` to each element of an array `A` to yield a new array via
-`f(A)`.   This kind of syntax is convenient for data processing, but in other languages vectorization
+`f(A)`. This kind of syntax is convenient for data processing, but in other languages vectorization
 is also often required for performance: if loops are slow, the "vectorized" version of a function
-can call fast library code written in a low-level language.   In Julia, vectorized functions are
+can call fast library code written in a low-level language. In Julia, vectorized functions are
 *not* required for performance, and indeed it is often beneficial to write your own loops (see
-[Performance Tips](@ref man-performance-tips)), but they can still be convenient.  Therefore, *any* Julia function
+[Performance Tips](@ref man-performance-tips)), but they can still be convenient. Therefore, *any* Julia function
 `f` can be applied elementwise to any array (or other collection) with the syntax `f.(A)`.
+For example `sin` can be applied to all elements in the vector `A`, like so:
+
+```jldoctest
+julia> A = [1.0, 2.0, 3.0]
+3-element Array{Float64,1}:
+ 1.0
+ 2.0
+ 3.0
+
+julia> sin.(A)
+3-element Array{Float64,1}:
+ 0.841471
+ 0.909297
+ 0.14112
+```
 
 Of course, you can omit the dot if you write a specialized "vector" method of `f`, e.g. via `f(A::AbstractArray) = map(f, A)`,
-and this is just as efficient as `f.(A)`.   But that approach requires you to decide in advance
+and this is just as efficient as `f.(A)`. But that approach requires you to decide in advance
 which functions you want to vectorize.
 
 More generally, `f.(args...)` is actually equivalent to `broadcast(f, args...)`, which allows
 you to operate on multiple arrays (even of different shapes), or a mix of arrays and scalars (see
-[Broadcasting](@ref)).  For example, if you have `f(x,y) = 3x + 4y`, then `f.(pi,A)` will return
+[Broadcasting](@ref)). For example, if you have `f(x,y) = 3x + 4y`, then `f.(pi,A)` will return
```

Comment:
That is like 50/50 in the manual... I have left those things untouched mostly though.